apps/builtin/README.md

```diff
@@ -0,0 +1,43 @@
+# builtin
+
+The **builtin** supports registering a TASH command through Makefile without codes change.  
+It means this only needs when TASH is enabled.
+
```

Comment:
I would suggest replacing this line with "It is needed only when TASH is enabled".